docs/encryption.md

```diff
@@ -0,0 +1,118 @@
+# containerd image encryption
+
+The containerd encryption feature allows the encryption and decryption of a container image.
+It is based on a [proposal for an OCI specification](https://github.com/opencontainers/image-spec/issues/747) to define the structure of an encrypted image.
+The encryption is specified on the layer, allowing the encryption of specific layers within an image. For example, given an image with an ubuntu, node and some custom code on top, only the custom code (top-most layer) can be encrypted to still benefit from layer deduplication of insensitive data. The key sharing is done via wrapped keys in the metadata.
+Therefore, an encrypted image data intended for multiple recipients can also be deduplicated.
+All this is done via the addition of additional layer mediatype `+enc`.
+More details can be viewed in the [design doc](https://docs.google.com/document/d/146Eaj7_r1B0Q_2KylVHbXhxcuogsnlSbqjwGTORB8iw).
+
+The two main usage points are in the creation of an encrypted image, and the decryption of the image upon usage.
+As most of the integration points would be transparent or consumed by other containerd runtime components or other utilities like buildkit, we have created several `ctr` commands to better illustrate the usage of the encryption feature.
+
+# Example End User Usage
+
+We have added 3 commands in the [`ctr`](https://github.com/containerd/containerd/tree/master/cmd/ctr) client under the image module. They are `ctr image encrypt/decrypt/layerinfo`.
```

Comment:
Enumerate the commands, rather than listing them as slashed items.

Reply:
Fixed cda9d21a57c541e5b2692f1b0f8d29826fc4daec